kube/pod.go

```diff
@@ -204,9 +204,19 @@ func getEnvVars(role *model.Role, defaults map[string]string, secrets SecretRefM
 			continue
 		}
 
-		ok, stringifiedValue := configValue(config, defaults)
-		if !ok {
-			continue
+		var stringifiedValue string
+		if createHelmChart {
+			required := ""
+			if config.Required {
+				required = fmt.Sprintf(`required "A valid .Values.env.%s is required" `, config.Name)
```

Comment:
Yes, we can make it whatever we want. But once we implement the new proposal and split the config settings into multiple namespaces, we will have to include the namespace part again.

I'll remove the prefix now since both of you seem to be bothered by it, but note that it will not improve the actual error message that much:

```
Error: render error in "uaa/templates/secret-1.yml": template: uaa/templates/secret-1.yml:10:30: executing "uaa/templates/secret-1.yml" at <required "A valid .V...>: error calling required: A valid .Values.env.UAA_ADMIN_CLIENT_SECRET is required
```

then becomes

```
Error: render error in "uaa/templates/secret-1.yml": template: uaa/templates/secret-1.yml:10:30: executing "uaa/templates/secret-1.yml" at <required "UAA_ADMIN...>: error calling required: UAA_ADMIN_CLIENT_SECRET configuration missing
```